### PR TITLE
Added my repo in Simple but innovative

### PR DESCRIPTION
**Pull request by Anirudhk07**
## What type of PR is this? (check all applicable)

- [x] 🚀 Added Name
- [ ] ✨ Feature
- [ ] ✅ Joined Community
- [ ] 🌟 ed the repo
- [ ] 🐛 Grammatical Error
- [ ] 📝 Documentation Update
- [ ] 🚩 Other

## Description

Added my repo in Simple but innovative

## Add Link of GitHub Profile

https://github.com/Anirudhk07

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,6 +76,7 @@
 - [Nguyễn Hoàng Dương](https://github.com/you-create/you-create)
 
 #### Simple but Innovative Ones 🤗
+- [Anirudh Kolwadkar](https://github.com/Anirudhk07/Anirudhk07)
```
Comment (abhisheknaiidu) [inline README.md:79]: Add your name in the last!

**Comment by Anirudhk07** [comment]
Hey ,
Can you please add it in your commits --

Name - Anirudh Kolwadkar
Github -https://github.com/Anirudhk07/Anirudhk07

File: README.md (hunk 1 above)
Comment (Anirudhk07) [reply README.md]: I have done the changes ! Kindly Review !

**Comment by Anirudhk07** [comment]
@abhisheknaiidu  review pls

**Pull request closed without merging**